content/en/service_management/case_management/customization.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Customization
+further_reading:
+- link: "service_management/case_management/"
+  tag: "Documentation"
+  text: "Case Management Overview"
+- link: "service_management/case_management/create_case"
+  tag: "Documentation"
+  text: "Create a case"
+- link: "service_management/case_management/settings"
+  tag: "Documentation"
+  text: "Settings"
+---
+
+## Overview
+
+Datadog Case Management allows customization to align with your team's unique workflows, data capture needs, and reporting requirements.
+
+## Custom Case Types
+
+<div class="alert alert-warning">
+  You must have Case Shared Settings Write (<code>cases_shared_settings_write</code>) permissions. For more information, see
+  <a href="https://docs.datadoghq.com/account_management/rbac/permissions/#case_management">Datadog Role Permissions</a>.
+</div>
+
+Datadog provides five [built-in case types][1], each designed for common workflows. To customize Case Management for your team's needs, you can define your own custom case types. This allows you to:
+
+* Scope custom data capture to relevant work types
+* Enable targeted automation
+* Conduct more granular analytics and reporting
+
+##### Create a custom case type
+
+1. Navigate to [**Settings > Shared Settings > Case Types**][2].
+2. Click **+ Create Case Type**.
```

Comment:
```suggestion
2. Click **+ Create Case Type**.
```
At first I didn't see the `+ Create Case Type`  - This is what I see on that screen: (with the + Create) at the bottom disabled because I don't have the right permissions ! That's what I get for not reading the callout right above :) 

<img width="1214" height="695" alt="image" src="https://github.com/user-attachments/assets/74363863-09d9-4d3b-8317-bde36a689af6" />